docs/projects/adv7513/index.rst

```diff
@@ -0,0 +1,181 @@
+A/V receivers, gaming consoles, and PCs.
+
+The digital video interface contains an HDMI v1.4/DVI v1.0-compatible transmitter
+and supports all HDTV formats. The :adi:`ADV7513` supports HDMI v1.4-specific
+features, including 3D video. The :adi:`ADV7513` also supports x.v.Color™,
+high bit rate (HBR) audio, and the programmable auxiliary video information
+(AVI) InfoFrame features. With the inclusion of HDCP, the :adi:`ADV7513` allows
+the secure transmission of protected content as specified by the HDCP v1.4 protocol.
+
+The :adi:`ADV7513` supports both S/PDIF and 8-channel I2S audio. Its high
+fidelity 8-channel I2S interface can transmit either stereo or 7.1 surround
+audio up to 768 kHz. The S/PDIF interface can carry compressed audio, including
+Dolby® Digital, DTS®, and THX®. Fabricated in an advanced CMOS process, the
+:adi:`ADV7513` is provided in a 64-lead LQFP surface-mount plastic package
+with exposed pad and is specified over the −25°C to +85°C temperature range.
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`ADV7513`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  Internal component of :intel:`De10-Nano <content/www/us/en/developer/topic-technology/edge-5g/hardware/fpga-de10-nano.html>`
+
+Block design
+-------------------------------------------------------------------------------
+
+The reference design uses HDL-related HDMI cores in order to interface the :adi:`ADV7513`
+IC into the De10-Nano evaluation kit.
+
+ADV7513 Block Diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: adv7513_bd.svg
+   :width: 800
+   :align: center
+   :alt: ADV7513 block diagram
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL(see more at :ref:`architecture`).
+
+========================  ===========
+Instance                  DE10-Nano
+========================  ===========
+sys_spi                   0x0108_0000
+sys_id                    0x0001_0000
+sys_gpio_bd               0x0001_0080
+sys_gpio_in               0x0001_0100
+video_dmac                0x0008_0000
+axi_hdmi_tx               0x0009_0000
+pixel_clk_pll_reconfig    0x0010_0000
+sys_gpio_out              0x0010_9000
+ltc2308_spi               0x0010_A000
+axi_sysid                 0x0001_8000
+========================  ===========
+
+GPIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Software GPIO number is calculated as follows:
+
+-  Altera Cyclone V SoC
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+   * -
+     - (from FPGA view)
+     -
+     - DE10-Nano
```

Comment:
Error when building the documentation, regarding this extra column (line 88) that was not removed from row 2. Plus the width of the table

```suggestion
   :widths: 40 30 30
   :header-rows: 2

   * - GPIO signal
     - Direction
     - HDL GPIO EMIO
   * -
     - (from FPGA view)
     - DE10-Nano
```